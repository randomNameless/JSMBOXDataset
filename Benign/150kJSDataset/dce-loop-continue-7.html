<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>dce-loop-continue-7.html</title>
</head>
<body>

<script>
for (;;) {
    if (cond) continue;
    continue;
}

</script>

</body>
</html>
