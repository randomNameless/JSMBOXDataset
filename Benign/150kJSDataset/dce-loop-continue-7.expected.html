<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>dce-loop-continue-7.expected.html</title>
</head>
<body>

<script>
for(;;)cond

</script>

</body>
</html>
